src/components/Tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { Box } from '../';
+import { Tab, TabList, TabPanel, Tabs } from './';
+
+export default {
+  component: Tabs,
+  title: 'Components/Tabs',
+};
+
+const TemplateTabs: ComponentStory<typeof Tabs> = (args: any) => {
+  return (
+    <Box width="100%">
+      <Box maxWidth="400px">
+        <Tabs {...args}>
```

Comment:
this is not looking proper. can we fix it ?

Reply:
Done

Reply:
Fixed